Submitted By: David B. Cortarello (Nomius) <nomius at users dot berlios dot de>
Date: 13-08-2008
Initial Package Version: 0.7.20
Description: The patch was created from the original mpd package
This patch provides the following features:
    * Fix infinite loop caused by continue sentence out of frame.

diff -Naur gtkdialog-0.7.20.orig/src/variables.c gtkdialog-0.7.20/src/variables.c
--- gtkdialog-0.7.20.orig/src/variables.c	2007-03-16 11:25:48.000000000 -0300
+++ gtkdialog-0.7.20/src/variables.c	2008-08-13 23:13:32.000000000 -0300
@@ -627,9 +627,10 @@
 			itemlist = GTK_LIST(actual->Widget)->children;
 			n = 0;
 			while (itemlist != NULL) {
-				if (itemlist->data == NULL) 
+				if (itemlist->data == NULL) {
 					itemlist = itemlist->next;
 					continue;
+				}
 				text = gtk_object_get_user_data(itemlist->data);
 				if (n == 0)
 					tmp = g_strconcat(line, "'", text, "'", NULL);
